test/std-test.cc

```diff
@@ -95,6 +96,28 @@ TEST(std_test, optional) {
 #endif
 }
 
+// this struct exists only to force a variant to be
+// valueless by exception
+// NOLINTNEXTLINE
+struct throws_on_move_t {
```

Comment:
`_t` suffix is normally used for aliases (typedefs), please remove.

Reply:
Fixed